app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -249,6 +253,8 @@ class TabsTrayFragment : AppCompatDialogFragment(), TabsTrayInteractor {
 
     override fun setCurrentTrayPosition(position: Int, smoothScroll: Boolean) {
         tabsTray.setCurrentItem(position, smoothScroll)
+        tab_layout.getTabAt(position)?.select()
+        tabsTrayStore.dispatch(TabsTrayAction.PageSelected(Page.positionToPage(position)))
```

Comment:
Can we move this as a method of the `TabLayoutMediator`? It would be nicer to have the logic that sync's the two layout types in one place.

Reply:
Agreed.  Thanks